microsoftgraph/microsoft-graph-explorer-v4
# Fix: Corrects position of feedback button and GE title

## Overview
Adds the feedback button to the minimised sidebar
Corrects the position of the feedback button and Graph Explorer title

### Demo
![image](https://user-images.githubusercontent.com/45680252/155479778-85947ee7-7cbf-41d1-b763-60ba40ef3d9d.png)

![image](https://user-images.githubusercontent.com/45680252/155479823-36a61b2a-cf7a-4119-8219-21b7da178ee7.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions
Check the app title section. Notice that the GE title, feedback button and hambugger menu are on the same line
Click on hambugger menu. Notice that the feedback button is on the menu
